tools/update_whatsnew.py

```diff
@@ -38,7 +39,11 @@ def getmtime(f):
 features, incompats = [], []
 for path in files:
     with io.open(path, encoding='utf-8') as f:
-        content = f.read().rstrip()
+        try:
+            content = f.read().rstrip()
+        except Exception as e:
+            raise Exception('Error reading "{}"'.format(f))
```

Comment:
This adds one bit of information - what file the error was on - but removes another: what error occurred.

Maybe print the filename and then reraise the original error?